ui/src/components/WMWorkerPoolEditor/index.jsx

```diff
@@ -286,6 +286,21 @@ export default class WMWorkerPoolEditor extends Component {
       onDialogActionComplete,
     } = this.props;
     const { workerPool, error, actionLoading, validation } = this.state;
+    const {
+      description,
+      emailOnError,
+      owner,
+      providerId,
+      workerPoolId,
+    } = this.props.workerPool;
+    const isWorkerPoolDirty =
+      isNewWorkerPool ||
+      workerPool.description !== description ||
+      workerPool.emailOnError !== emailOnError ||
+      workerPool.owner !== owner ||
+      workerPool.providerId !== providerId ||
+      joinWorkerPoolId(workerPool.workerPoolId1, workerPool.workerPoolId2) !==
+        workerPoolId;
 
```

Comment:
STR:

1. Navigate to http://localhost:5080/worker-manager and make sure you are logged out
2. Click on any worker pool id that's under "Worker Pool ID"
3. Modify the code editor under "Configuration"

Expected: The save button becomes enabled.
Actual: Nothing happens.

Reply:
My bad .Problem resolved , Have a look .
@helfi92